portfolio/src/main/webapp/index.html

```diff
@@ -2,15 +2,30 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Ricardo Paz's Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
+      <h1>Ricardo Paz's Portfolio</h1>
+      <p>I am a student from Michigan State University. I am studying 
+          Computer Science with a minor in Cognitive Science. My interest
+          with Computer Science began when many companies had data leaks. 
+          I am interested in learning about Cyber Security and how to create a 
+          more secure digital world.
+      </p>
+      <p>I am a First Generation Latino Student as well. I am a Mexican Honduran American.
+          I love my culture and try to show others why I am who I am. 
+      </p>
+      <p> LinkedIn Profile: www.linkedin.com/in/pazricar
+          GitHub Profile: in the process
+      </p>
+      <p> I am currently very involved on campus. I am the Secretary for the Largest Student 
+          organization on campus. It is called Culturas de las Razas Unidas. CRU for short.
+          I am also part of the Vice President and Chief Diversity Officer Search Committee for 
+          Michigan State University. 
+      <img src="images/Luis7.JPG" style="width:500px;height:400px;/>
```

Comment:
I think you need to commit `images/Luis7.JPG` as well, otherwise I won't be able to see it :)

Reply:
I am gonna need some help with commiting, still kinda new to it.

Reply:
So assuming that you added your image according to [these instructions](https://github.com/pazricardo/software-product-sprint-or-my-portfolio/blob/master/walkthroughs/week-1-web-development/portfolio-walkthrough.md#uploading-images), then you should probably run:

`git add images/Luis7.JPG`
`git commit`